spec/crc_spec.rb

```diff
@@ -6,14 +6,6 @@
     it { expect(subject.block_length).to be 1 }
   end
 
-  describe ".pack" do
-    subject { described_class }
-
-    it do
-      expect { subject.pack(0) }.to raise_error(NotImplementedError)
-    end
-  end
-
```

Comment:
There should be a generic `CRC.pack` spec in `spec/crc_examples.rb`, even though `.pack` is called in `#finish` which is used by `#hexdigest`/`.hexdigest`. There's a lot of nuanced logic in `CRC.pack` which needs specs.

Reply:
Is this a sufficient change?